.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,11 +1,20 @@
+<!-- !! Thank your for opening a PR !! -->
 
-**Thank your for opening a PR!**
+**What is this PR about?**
+...
 
-Before you proceed, please make sure that [the pre-commit linting/style checks pass](https://docs.pymc.io/en/latest/contributing/python_style.html).
+**Checklist**
++ [ ] Explain important implementation details 👆
++ [ ] Make sure that [the pre-commit linting/style checks pass](https://docs.pymc.io/en/latest/contributing/python_style.html).
++ [ ] Link relevant issues (preferably in [nice commit messages](https://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html))
++ [ ] Are the changes covered by tests and docstrings?
++ [ ] Fill out the release notes sections 👇
```

Comment:
I thought we dropped this?

Reply:
Just the sections in the PR description. To make it easier to quickly scan the impact a PR has

Reply:
```suggestion
+ [ ] Fill out the short summary sections 👇
```